source/cloud/azure/azureml.md

```diff
@@ -55,9 +65,147 @@ Launch the instance.
+2. Find the file/folder containing your dataset and click the elipsis (...) next to it.
+3. From the menu, choose **Copy URI** and select **Datastore URI** format to copy into your notebook.
+
+![Screenshot of access datastore uri screen](../../images/azureml-access-datastore-uri.png)
+
+### Custom RAPIDS Environment
+
+To run an AzureML experiment, you must specify an [environment](https://learn.microsoft.com/en-us/azure/machine-learning/concept-environments?view=azureml-api-2) that contains all the necessary software dependencies to run the training script on distributed nodes. <br>
+You can define an environment from a [pre-built](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-docker-image) docker image or create-your-own from a [Dockerfile](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-docker-build-context) or [conda](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-conda-specification) specification file.
+
+Create your custom RAPIDS docker image using the example below, making sure to install additional packages needed for your workflows.
+
+```dockerfile
+
+# Use latest rapids image with the necessary dependencies
+FROM {{ rapids_container }}
+
+# Update and/or  install required packages
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential fuse && \
+    rm -rf /var/lib/apt/lists/*
+
+# Activate rapids conda environment
+RUN /bin/bash -c "source activate rapids && pip install azureml-mlflow"
+```
+
+Now create the Environment, making sure to label and provide a description:
+
+```python
+from azure.ai.ml.entities import Environment, BuildContext
+
+env_docker_image = Environment(
+    build=BuildContext(path="Dockerfile"),
+    name,
+    description,
+)
+
+ml_client.environments.create_or_update(env_docker_image)
+```
 
-Then in JupyterLab select the `rapids` kernel when working with a new notebook
+### Submit RAPIDS Training jobs
+
+Now that we have our environment and custom logic, we can configure and run the `command` [class](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml?view=azure-python#azure-ai-ml-command) to submit training jobs. `inputs` is a dictionary of command-line arguments to pass to the training script.
+
+```python
+from azure.ai.ml import command, Input
+
+# submit training job
+command_job = command(
+    environment,
+    experiment_name,
+    code,
+    command,
+    inputs,
+    compute,
+)
+returned_job = ml_client.jobs.create_or_update(command_job)
+
+
+# apply hyperparameter sweep_job
+sweep_job = command_job.sweep(
+    compute,
+    sampling_algorithm,
+    primary_metric,
+    goal="Maximize",
+)
```

Comment:
Again this code is not valid, none of these variables exist and I think they are kwargs rather than args anyway.

We should have a small and complete example here of running a job. Even if it just a small hello world script.